website/content/en/preview/concepts/disruption.md

```diff
@@ -118,7 +118,10 @@ Using preferred anti-affinity and topology spreads can reduce the effectiveness
 {{% /alert %}}
 
 {{% alert title="Note" color="primary" %}}
-For spot nodes, Karpenter only uses the deletion consolidation mechanism.  It will not replace a spot node with a cheaper spot node.  Spot instance types are selected with the `price-capacity-optimized` strategy and often the cheapest spot instance type is not launched due to the likelihood of interruption. Consolidation would then replace the spot instance with a cheaper instance negating the `price-capacity-optimized` strategy entirely and increasing interruption rate.
+For spot nodes, Karpenter has deletion consolidation enabled by default. If you would like to enable replacement with spot consolidation, you need to enable the feature through the [`SpotToSpotConsolidation` feature flag]({{<ref "../reference/settings#features-gates" >}}).
```

Comment:
Looking over the rendered documentation, this has become less of a "note" and more of its own section. Can you just create a sub-section under consolidation at this point for spot consolidation? Looks like it should be an h4 header